src/components/mainpagecard/card1.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+const Card1 = (props) => {
+  return (
+        <div className="card card-component">
+            <img className="card-img" src={props.img}/>
```

Comment:
Suggestion: provide `alt` attribute with `img` tag, it will help in accessibility

Reply:
Will do